service/gradle/dependencies.gradle

```diff
@@ -43,14 +43,11 @@ dependencies {
   // hk2 is required to use datarepo client, but not correctly exposed by the client
   implementation group: "org.glassfish.jersey.inject", name: "jersey-hk2"
 
-  // OpenTelemetry
-  // Versioned by Spring Boot Dependency Manager via opentelemetry-bom:
-  implementation 'io.opentelemetry:opentelemetry-api'
```

Comment:
We can remove this dependency because ersion of TCL exposes this dependency at the api level, and Spring Boot dependency manager includes and versions the `opentelemetry-bom`.

https://github.com/DataBiosphere/terra-common-lib/pull/164/files